To: vim_dev@googlegroups.com
Subject: Patch 7.3.228
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.228
Problem:    "2gj" does not always move to the correct position.
Solution:   Get length of line after moving to a next line. (James Vega)
Files:      src/normal.c


*** ../mercurial/vim73/src/normal.c	2011-05-10 17:29:28.000000000 +0200
--- src/normal.c	2011-06-20 00:41:43.000000000 +0200
***************
*** 4533,4538 ****
--- 4533,4539 ----
  		}
  		curwin->w_cursor.lnum++;
  		curwin->w_curswant %= width2;
+ 		linelen = linetabsize(ml_get_curline());
  	    }
  	}
        }
*** ../vim-7.3.227/src/version.c	2011-06-20 00:25:37.000000000 +0200
--- src/version.c	2011-06-20 00:45:15.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     228,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
197. Your desk collapses under the weight of your computer peripherals.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
